### PR TITLE
fix: migration to bootstrap 5

### PR DESCRIPTION
**Pull request by bodnara**
Renamed the following classes
* left-* to start-*
* right-* to end-*
* ml-* to ms-*
* mr-* to me-*
* pl-* to ps-*
* pr-* to pe-*
* font-weight-* to fw-*

### CODE REVIEW

File: libs/perun/dialogs/src/lib/member-tree-view-dialog/member-tree-view-dialog.component.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
   <h1 mat-dialog-title>
     {{'SHARED_LIB.PERUN.COMPONENTS.MEMBER_TREEVIEW_DIALOG.TITLE' | translate:{ 'name': userName} }}
   </h1>
-  <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  <mat-spinner *ngIf="loading" class="ms-auto me-auto"></mat-spinner>
   <div *ngIf="!loading" mat-dialog-content class="mr-0">
```
Comment (mattjoke) [inline libs/perun/dialogs/src/lib/member-tree-view-dialog/member-tree-view-dialog.component.html:6]:
```suggestion
  <div *ngIf="!loading" mat-dialog-content class="ms-0">
```

File: libs/perun/components/src/lib/attribute-search-select/attribute-search-select.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div *ngIf="options !== undefined" class="container ml-0 mr-0 pl-0 pr-0 mt-3">
+<div *ngIf="options !== undefined" class="container ml-0 mr-0 ps-0 pe-0 mt-3">
```
Comment (mattjoke) [inline libs/perun/components/src/lib/attribute-search-select/attribute-search-select.component.html:1]:
```suggestion
<div *ngIf="options !== undefined" class="container ms-0 me-0 ps-0 pe-0 mt-3">
```

**Review by mattjoke — CHANGES_REQUESTED** [review]
Just skimmed through the files, however I think someone else needs also to look at this.

File: apps/publications/src/app/pages/create-publication-page/import-publications-page/import-publications-page.component.html [hunk 3]
```diff
@@ -22,19 +22,19 @@ <h1 class="page-subtitle d-flex">
       <perun-web-apps-year-range [startYear]="startYear" [endYear]="endYear">
       </perun-web-apps-year-range>
 
-      <button mat-stroked-button class="left-space" (click)="searchPublications()">
+      <button mat-stroked-button class="start-space" (click)="searchPublications()">
```
Comment (HejdaJakub) [inline apps/publications/src/app/pages/create-publication-page/import-publications-page/import-publications-page.component.html:25]: Again incorrect replacement. In this class is more similar cases like that for `start-space` and also for `end-button`.

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-application-form/vo-settings-application-form.component.html [hunk 4]
```diff
@@ -78,7 +78,7 @@ <h1 class="page-subtitle">
     <!--    <button-->
     <!--      *ngIf="editAuth"-->
     <!--      (click)="clear()"-->
-    <!--      class="ml-2"-->
+    <!--      class="ms-"-->
```
Comment (HejdaJakub) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-application-form/vo-settings-application-form.component.html:81]: I know it's just a comment, but for consistency.
```suggestion
    <!--      class="ms-2"-->
```

**Review by HejdaJakub — CHANGES_REQUESTED** [review]

**Comment by bodnara** [comment]
- reverted changes to custom classes
- removed decorations on links
- aligned user name and menu in top header
- set proper display of icons inside button (refresh, back, ...)

**Review by HejdaJakub — APPROVED** [review]

**Pull request merged**